pkg/config/linters.go

```diff
@@ -9,3 +14,52 @@ type Linters struct {
 
 	Presets []string
 }
+
+func (l *Linters) Validate() error {
+	if err := l.validateAllDisableEnableOptions(); err != nil {
+		return err
+	}
+
+	if err := l.validateDisabledAndEnabledAtOneMoment(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *Linters) validateAllDisableEnableOptions() error {
+	if l.EnableAll && l.DisableAll {
+		return errors.New("--enable-all and --disable-all options must not be combined")
+	}
+
+	if l.DisableAll {
+		if len(l.Enable) == 0 && len(l.Presets) == 0 {
```

Comment:
cosmetic: gocritic also highlights that linters were already enabled or disabled via presets

Reply:
gocritic handles its own rules, not our linters or presets.

Reply:
of course, I meant checkers 
just for analogy

Reply:
So I don't understand what is your suggestion here.